src/lib/remark-plugins/rewrite-static-assets.ts

```diff
@@ -15,7 +15,11 @@ import { Plugin } from 'unified'
 import { Node } from 'unist'
 import { Image, Definition } from 'mdast'
 
-const ASSET_API_ENDPOINT = `${process.env.MKTG_CONTENT_API}/api/assets`
+// This env is set for local docker previews by a custom asset server,
+// otherwise we use the content api for previews / prod
+const ASSET_API_ENDPOINT =
+	process.env.ASSET_API_ENDPOINT ||
+	'https://mktg-content-api-hashicorp.vercel.app/api/assets'
```

Comment:
Should we use `${process.env.MKTG_CONTENT_API}/api/assets` for the fallback?

Reply:
I think in dev portal sometimes the link is `content.hashicorp.com` instead of `mktg-content-api-hashicorp.vercel.app`

Reply:
Oh yes, good catch. thank you!